code/modules/integrated_electronics/subtypes/input.dm

```diff
@@ -586,11 +586,12 @@
 	desc = "Enables the sending and receiving of messages on NTNet via packet data protocol."
 	extended_desc = "Data can be send or received using the second pin on each side, \
 	with additonal data reserved for the third pin. When a message is received, the second activation pin \
-	will pulse whatever's connected to it. Pulsing the first activation pin will send a message."
+	will pulse whatever's connected to it. Pulsing the first activation pin will send a message. Message \
+	can be send to multiple recepients. Addresses must be separated with ; symbol."
```

Comment:
Just make the target NTNet addresses pin a list pin.

Reply:
so you should fuck with lists every time you need send message to one adress?
Better to fuck with list concatenator.

Reply:
By the way.Are you going to reopen your PR with cameras?